test/unit/util_spec.js

```diff
@@ -185,6 +185,23 @@ describe("util", function () {
       const str = "string\x00With\x00Null\x00Chars";
       expect(removeNullCharacters(str)).toEqual("stringWithNullChars");
     });
+
+    it("should modify string with non-displayable characters", function () {
+      const str = Array.from(Array(32).keys())
+        .map(x => String.fromCharCode(x) + "a")
+        .join("");
+      // \x00 is replaced by an empty string.
+      const expected =
+        "a" +
+        " "
+          .repeat(31)
+          .split("")
+          .map(x => x + "a")
+          .join("");
```

Comment:
Given that it's the expected value, I cannot help wondering if simply spelling it out explicitly wouldn't actually be clearer and easier to read?
```javascript
const expected =
  "a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a";